plugins/ftdidmx/Makefile.mk

```diff
@@ -13,7 +13,14 @@ plugins_ftdidmx_libolaftdidmx_la_SOURCES = \
     plugins/ftdidmx/FtdiWidget.cpp \
     plugins/ftdidmx/FtdiWidget.h
 plugins_ftdidmx_libolaftdidmx_la_LIBADD = \
-    $(libftdi_LIBS) \
     common/libolacommon.la \
     olad/plugin_api/libolaserverplugininterface.la
+if HAVE_LIBFTDI1
+plugins_ftdidmx_libolaftdidmx_la_LIBADD += \
+    $(libftdi1_LIBS)
+else
+plugins_ftdidmx_libolaftdidmx_la_LIBADD += \
+    $(libftdi_LIBS)
```

Comment:
I think this wants to be a zero now given I/we changed this line to be libftdi0 from libftdi:
https://github.com/OpenLightingProject/ola/pull/1422/files#diff-67e997bcfdac55191033d57a16d1408aR624

Reply:
Fixed